rs/ledger_suite/icrc1/ledger/Cargo.toml

```diff
@@ -48,10 +49,13 @@ ic-test-utilities-load-wasm = { path = "../../../test_utilities/load_wasm" }
 leb128 = "0.2.4"
 num-bigint = { workspace = true }
 proptest = { workspace = true }
+ic-certification = { workspace = true }
```

Comment:
nit: These (and the other added dependencies) currently don't follow lexographical ordering.